perun-utils/ldapc-scripts/schemas/perun-schema.ldif

```diff
@@ -59,9 +59,10 @@ olcAttributeTypes: {54}( 1.3.6.1.4.1.8057.2.80.60 NAME 'aup' DESC 'JSON represen
 olcAttributeTypes: {55}( 1.3.6.1.4.1.8057.2.80.61 NAME 'isAssignedWithSubgroups' DESC 'True if all subgroups of assigned groups should be considered as assigned too.' EQUALITY caseIgnoreMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE )
 olcAttributeTypes: {56}( 1.3.6.1.4.1.8057.2.80.62 NAME 'tcsMails' DESC 'List of users mail addresses used for TCS' EQUALITY caseIgnoreIA5Match SUBSTR caseIgnoreIA5SubstringsMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.26{256} )
 olcAttributeTypes: {57}( 1.3.6.1.4.1.8057.2.80.63 NAME 'sshPublicKey' DESC 'User`s public SSH key.' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
+olcAttributeTypes: {58}( 1.3.6.1.4.1.8057.2.80.64 NAME 'sponsoredMembershipInOrganizations' DESC 'List of sponsored memberships in organizations.' EQUALITY caseIgnoreMatch SUBSTR caseIgnoreSubstringsMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
```

Comment:
Please make this ``caseExactMatch`` and ``caseExactSubstringsMatch``. Thank you.